aiida/backends/manager.py

```diff
@@ -114,13 +114,18 @@ def get_settings_manager(self):
         :return: `SettingsManager`
         """
 
-    def load_backend_environment(self, profile, validate_schema=True, **kwargs):
+    def load_backend_environment(self, profile, validate_schema=True, verify_lock=True, **kwargs):
```

Comment:
As mentioned also below - let's not make this variable available to users, unless we have a very strong good reason to do so